packages/Metric/src/Score.tsx

```diff
@@ -36,6 +44,29 @@ const Score: React.FunctionComponent<ScoreProps> = ({
 }: ScoreProps) => {
     const stringFormatter = useLocalizedStringFormatter(intlMessages);
     const { locale } = useLocale();
+
+    const getBrand = (): string => {
```

Comment:
sorry I didn't notice this before. I usually put this function outside of Score function component.